00_Core/Scripts/Source/MilkQUEST.psc

```diff
@@ -594,9 +599,18 @@ Function MilkCycle(Actor akActor, int t)
 
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.LactacidCount", LactacidCnt)
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.MilkGen", MaidMilkGen)
-	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.MilkCount", MilkCnt)
+	MME_Storage.setMilkCurrent(akActor, MilkCnt)
 	StorageUtil.SetFloatValue(akActor,"MME.MilkMaid.PainCount", PainCnt)
 
+	; setMilkCurrent() automatically restricts the provided value to the maximum allowed value
```

Comment:
Can you help me out with https://github.com/ThomasNeumann/Milk-Mod-Economy/commit/33b48a64f2ce9236ff8751be7e367c2cd50e575b ? I'm having trouble to reference a property from MilkQuest. I added the following lines to be able to access the global limit
```
MilkQUEST Property MilkQ Auto
[...]
return MilkQ.BreastScaleLimit
```
but PapyrusCompiler complains

> MME_Storage.psc(175,8): variable MilkQ is undefined
> MME_Storage.psc(175,14): none is not a known user-defined type

I tried to do exactly as in MilkMCM.psc, but something seems to be missing.